crates/api_models/src/payments.rs

```diff
@@ -3663,6 +3664,20 @@ pub enum PaymentIdType {
     PreprocessingId(String),
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize, ToSchema)]
+#[cfg(feature = "v2")]
+pub enum PaymentIdType {
+    /// The identifier for payment intent
+    PaymentIntentId(id_type::GlobalPaymentId),
+    /// The identifier for connector transaction
+    ConnectorTransactionId(String),
+    /// The identifier for payment attempt
+    PaymentAttemptId(String),
```

Comment:
which are connectors and flows we need `PaymentAttemptId` for processing webhooks?

Reply:
can you change this to `GlobalAttemptId`?